ToDoApp/ToDoApp.xcodeproj/project.pbxproj

```diff
@@ -7,7 +7,10 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
-		3A4C5F152938393A002EEE92 /* ToDoService.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3A4C5F142938393A002EEE92 /* ToDoService.swift */; };
```

Comment:
let's keep files organized and create a folder structure that would be easier to read.
First let's drop content view file since it is not used anymore
Then let's create a Services folder where the ToDoService file will go
Then let's create a CreateTaskFolder where CreateTaskView and CreateTaskViewModel will go

Reply:
This has been fixed.